### DISCUSSION TITLE
@mouseover @@mouseleave event add in petite-vue

### DISCUSSION THREAD

**Discussion by shahensaifullah** [vuejs/petite-vue#152] [Ideas]
@mouseover and @mouseleave are not available in petite vue. Is there any way to use these in my petite vue?

**Comment by fparedlo** [comment]
Yoy can wrap all your event listeners in a function and run this inside the
mounted event. No need to use vue events to create normal js events.

**Reply by shahensaifullah** [reply]
could you please give me a short example?

**Reply by sqllyw** [reply]
https://jsfiddle.net/9zeswu8L/

**Reply by fparedlo** [reply]
> could you please give me a short example?

sorry took me some time to come back to you. Example with click event https://codesandbox.io/s/determined-matsumoto-1rskx0?file=/index.html

**Comment by celeg** [comment]
or based with exemple of sqllyw 
Replace js

`<p @vue:mounted="mounted($el)" @vue:unmounted="unmounted($el)">{{ plusOne }} mouse over</p>`
By vue : 
`<p @mouseover="increment" @mouseleave="console.log('bye')">{{ plusOne }} mouse over</p>`
